ideas/1XX-nail-swarm-process.md

```diff
@@ -0,0 +1,115 @@
+## Preamble
+
+    Idea: 1XX-nail-swarm-process
+    Title: Nail swarm process
+    Status: Draft
+    Created: 2018-03-30
+    Requires (*optional):
+    Replaces (*optional):
+
+## Summary
+Swarms is the primary structure we use to turn ideas into reality at Status.
+People who are excited about bringing an idea to fruition commit to it and make
+sure it happens.
+
+At an individual level, this ensures individual contributors are enabled to
+achieve autonomy, mastery and purpose, while still ensuring accountability.
+
+At a product team / organizational level, swarms allow us to see what is going
+on and which OKRs are targetted, which are not, and generally track progress at
+a reasonable level of granularity.
+
+However, the process for creating, running and completing swarms is currently
+ad-hoc, lacking in accountability and follow-up, and knowledge of how to do this
+properly isn't evenly spread throughout the organization. This swarm will fix
+this.
+
+## Vision
+
+There are two constraints that govern how this structure is designed and
+cultivated. One is that Status the company wants to ship a product, and the
+other is creating a decentralized autonomous organization. Since we are moving
+towards the latter, the structures we are building must be compatible with this
+latter vision, while still being mindful of the first.
+
+The primary difference between the two is that in the former, we are speaking on
+behalf of users, and people are generally compensated on a salary or salary-like
+basis.
+
+We thus distinguish between 'swarms' (type 1) more generally and 'Status LLC
+using swarms' (type 2), where the latter might have more constraints given its
+immediate goals and current compensation model. This allows us to be efficient
+and effective while the rest of the organization is being built up.
+
+For example, a reasonable requirement for a type 2 swarms is that they target
+OKRs and that PM/UX/Eng should approve an idea before it is given go ahead,
+whereas these requirement might not make sense for a type 1 swarm.
+
+## Swarm Participants
+- Lead Contributor: @oskarth
+- Evaluation ('knowledge spreading'): Voting mechanism (Polly survey)
+- Evaluation ('OKR coverage'): TBC: @chad and/or @nabil
+- Evaluation ('enable experimentation'): TBC: @arash and/or @martin
+- Contributor: @martin (TBC)
+
+Need one more contributor and evaluator, possibly PM. @naghdy or @jarradh as
+evaluator? Possibly UX for communicating things during Town Hall, etc.
+
+### Requirements & Dependencies
+
+### Minimum Viable Product
+Goal Date: 2017-04-06
+Description:
+i. Ensure new draft / in-progress ideas use PR review process.
+ii. Measure Core/Swarm OKR coverage and communicate this/needs.
+iii. Evaluation terminology and expectations are clarified in manual.
```

Comment:
Right now there's conflation on several dimensions:
- external (reality) vs internal self/group-evaluation
- as a person/role or as a mechanism
- as acting on behalf of something or someone (like users)

This has been talked about a lot in various contexts, but there's a confusion and lack of clarity around terminology, as well as difference between structural rules and customs (common law style).

Reply:
For type 1 swarms, this ties into discussion around proportional payouts (internal self-evaluation) and anon evaluation panels (external interface).